client/src/components/ColorSelector.jsx

```diff
@@ -1,12 +1,28 @@
 import React from 'react';
-import {ColorContainer} from '../styling.jsx';
+import ColorButton from './ColorButton.jsx';
+import {ColorContainer, RadioButtonLabel} from '../styling.jsx';
+import styled from 'styled-components';
 
 const ColorSelector = (props) => {
   return (
     <ColorContainer>
-      Color selector goes here
+
+      {props.colors.map((color) => {
+        return <ColorButton key={color} color={color} selectColor={props.selectColor}/>
```

Comment:
The nitpicking continues! 

This is pretty minor but I'm curious what your limit is before you start putting element attributes on new lines?

```
<ColorButton 
  key={color} 
  color={color} 
  selectColor={props.selectColor}
/>
```
If you're cool with keeping it inline, then a space before the closing `/>` is my suggestion (for readability).

Reply:
I personally like to make new lines once I reach 3 attributes, because it makes it easier to read when I haven't seen the code in a while. I lean on the side of spacing code a lot